.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+        node_version: [12, 13]
+    name: CI on NodeJS v${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - run: git config --global core.autocrlf false
```

Comment:
Avoid CR/LF issues when cloning in Linux/Windows with default git configurations, that causes linting erros (on Windows, git by default clones with CRLF, and then the linter use to check that it ends in LF and fails the CI)